terraform/modules/hetzner-kubernetes/main.tf

```diff
@@ -0,0 +1,27 @@
+# FUTUREWORK: Attach a volume for etcd state, so we can recreate this machine
+# when we need to.
+resource "hcloud_server" "node" {
+  count       = var.node_count
+  name        = "${ var.environment }-kubenode${ format("%02d", count.index + 1 )}"
+  image       = var.image
+  server_type = var.server_type
+  ssh_keys    = var.ssh_keys
+  location    = var.location
+
+  labels = {
+    # FUTUREWORK: This label name is very undecriptive and it should be renamed
```

Comment:
Followup ticket and link to it here?

Reply:
I don't think this kind of refactorings need a ticket. We have such FUTUREWORK comments in a quite a few places. They are useful to someone already making a breaking change to also do it in that place.